statistics/pooled_prevalence.R

```diff
@@ -0,0 +1,100 @@
+library(tidyverse)
+library(rstan)
+
+options(mc.cores = parallel::detectCores())
+# rstan_options(auto_write = TRUE)
+
+stan_model <- stan_model(model_code = "
+data {
+  // Pooled testing data
+  int<lower=0> N_pools;              // number of pools
+  int<lower=0> x[N_pools];           // test results (0 = negative, 1 = positive)
+  int<lower=1> n[N_pools];           // size of each pool
+  
+  // MGS data
+  int<lower=0> N_mgs;                // number of MGS samples
+  int<lower=0> viral_reads[N_mgs];   // observed viral reads in each MGS sample
+  int<lower=0> total_reads[N_mgs];   // total reads in each MGS sample
```

Comment:
Shouldn't matter at the scale we're at, but why `lower=0` for MGS samples -- can we have a 0-read sample?